pyproject.toml

```diff
@@ -72,7 +72,6 @@ pytest-cov = ">=4.0.0,<5.0"
 ruff = "^0.4.9"
 pandas = [
     {version = ">=2.1.1,<3.0", python = ">=3.9,<3.13"},
```

Comment:
We can remove the python version checks for pandas, pillow, and pre-commit below